exonum/src/events/noise/tests.rs

```diff
@@ -107,6 +112,84 @@ fn test_converted_keys_handshake() {
         .expect("Unable to transition session into transport mode");
 }
 
+#[test]
+fn noise_encrypt_decrypt_short_message() {
+    check_encrypt_decrypt_message(64);
```

Comment:
Please add a test case with

```rust
check_encrypt_decrypt_message(MAX_MESSAGE_LENGTH - 
    SIGNATURE_LENGTH -
    HEADER_LENGTH -
    10)
```

(`10` can be replaced with another small integer). Such a test fails as of now because of incorrect message length calculation (see another line comment).